src/animation.jl

```diff
@@ -80,7 +80,9 @@ function buildanimation(animdir::AbstractString, fn::AbstractString;
 
     catch err
         warn("""Tried to create gif using convert (ImageMagick), but got error: $err
-            ImageMagick can be installed by executing `Pkg.add("ImageMagick")`
+            ImageMagick needs to be installed in julia and on your machine:
+              1) in julia>> `Pkg.add("ImageMagick")`
+              2) on your machine (in terminal run): sudo apt install imagemagick
```

Comment:
there does not need to be a separate install for imagemagick on windows? i.e. you just need to run add the package in julia?